### PR TITLE
android: update uuid to get rid of the warning

### PR DESCRIPTION
**Pull request by 8BallBomBom**
Started getting this warning recently.

warning react-native-webrtc > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

Should be a simple non-disruptive upgrade.

### CODE REVIEW

**Comment by saghul** [comment]
> Should be a simple non-disruptive upgrade.

On the contrary. This requires `crypto.getRandomValues` to exist, which requires the app to use a polyfill: https://github.com/uuidjs/uuid#getrandomvalues-not-supported

Since we don't use this for cryptographic purposes, but just as a simple ID to map objects I think a better solution would be to inline a function that generated a "unsafe" UUID v4 with Math.random.

**Comment by 8BallBomBom** [comment]
Ahhhhh absolutely didn't notice that, hmmmmm...

**Comment by 8BallBomBom** [comment]
@saghul Alrighty, nothing fancy but replicates the UUID v4 format 😃

File: src/RTCUtil.js [hunk 1]
```diff
@@ -112,6 +112,14 @@ function normalizeMediaConstraints(constraints, mediaType) {
     }
 }
 
+function chr4() {
+    return Math.random().toString(16).slice(-4);
+}
+
+export function uniqueID() {
```
Comment (saghul) [inline src/RTCUtil.js:119]: Can you please add some docs here explaining why we are doing it this way?

**Review by saghul — COMMENTED** [review]
LGTM, please add some docs and we're good!

**Comment by 8BallBomBom** [comment]
I'll get to it, would you also prefer that i add the string together in a different way?

File: src/RTCUtil.js (hunk 1 above)
Comment (8BallBomBom) [reply src/RTCUtil.js:132]: Hopefully good enough 👍🏻

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Cheers!

**Pull request merged**